memtest/python/memtest/__main__.py

```diff
@@ -0,0 +1,37 @@
+"""CLI for lance-memtest."""
```

Comment:
How is this CLI intended to be used?  It seems more like a helper tool for debugging the library?

Reply:
Removed the stats command, as that's useless. It's mainly meant to get the library path so you can run:

```shell
LD_PRELOAD=$(lance-memtest) ...
```